packages/thirdweb/src/extensions/cryptopunks/__generated__/CryptoPunks/read/punkAttributes.ts

```diff
@@ -0,0 +1,130 @@
+import type { AbiParameterToPrimitiveType } from "abitype";
```

Comment:
Do we need a Punks extension? I'm starting to get concerned we're bloating the SDK

Reply:
Nope I'm removing them since both you and Jonas have the same idea